assets/js/main.js

```diff
@@ -0,0 +1,173 @@
+var gameArray = []
```

Comment:
Need `document.addEventListener('DOMContentLoaded', function() {})` here. Be aware that you need to make sure that your DOM is fully loaded first before you did anything.